graphium/data/multilevel_utils.py

```diff
@@ -22,6 +22,11 @@ def unpack(graph_data):
         elif isinstance(graph_data, list):
             return np.array(graph_data)
         elif isinstance(graph_data, np.ndarray):
+            if len(graph_data.shape) == 0:
+                graph_data = np.expand_dims(graph_data, 0)
+            if graph_data.shape[0] == 0:
+                graph_data = np.array([np.nan])
```

Comment:
This seems like a risky behaviour that could lead to a shape mismatch. In what cases is `graph_data` empty?

Reply:
I observed this when parsing the PCQM4M data and added the missing case. A shape mismatch shouldn't occur because later in the same function we do nan-padding to fit the shapes again. If you still think this could be an issue, please elaborate on why it could lead to a shape mismatch. Perhaps I don't fully understand what you mean.

Reply:
Is `graph_data.shape` giving you the number of nodes? 

If so, I know that there's a bug in our custom PCQM4M dataset where some molecules have a mismatched number of hydrogens, because hydrogens where always removed from the labels, but some molecules require explicit hydrogens (H2 for instance), which ended having 0 atoms.

Do you think it is related to that? If so, it's not the appropriate fix, but we could accept it temporarily.

Reply:
To be precise, `graph_data.shape` gives you the number of labels. So yes, when pre-processing the PCQM4M data, I encountered cases where no labels were given for the specific graph.

@zhiyil-graphcore Did you encounter any issues when pre-processing the LargeMix? Because if not, I'm confused why this caused issues on my side.

I can try to remove this again and see whether the error disappears with the newest changes in the main branch.